src/App.tsx

```diff
@@ -53,6 +63,16 @@ function App() {
       }
     : null;
 
+  const action = (snackbarId: SnackbarKey) => (
+    <IconButton
+      onClick={() => {
+        closeSnackbar(snackbarId);
+      }}
+    >
+      <CloseIcon className="text-white" />
+    </IconButton>
+  );
+
```

Comment:
nice - we probably want to also refactor this soon into it's own component though